modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-core-platform-alerting-rules_{context}"]
+= Modifying core platform alerting rules
+
+As a cluster administrator, you can modify core platform alerts before Alertmanager routes them to a receiver. 
+For example, you can change the severity label of an alert, add a custom label, or exclude an alert from being sent to Alertmanager.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have enabled Technology Preview features, and all nodes in the cluster are ready.
+
+
+.Procedure
+
+. Create a new YAML configuration file named `example-modified-alerting-rule.yaml` in the `openshift-monitoring` namespace.
+
+. Add an `AlertRelabelConfig` resource to the YAML file. 
+The following example modifies the `severity` setting to `critical` for the default platform `watchdog` alerting rule: 
++
+[source,yaml]
+----
+apiVersion: monitoring.openshift.io/v1alpha1
+kind: AlertRelabelConfig
+metadata:
+  name: watchdog
+  namespace: openshift-monitoring
+spec:
+  configs:
+  - sourceLabels: [alertname,severity] <1>
+    regex: "Watchdog;none" <2>
+    targetLabel: severity <3>
+    replacement: critical <4>
+    action: Replace <5>
+----
+<1> The source labels for the values to be modified.
+<2> The name of the alerting rule you want to modify.
+<3> The target label of the value you want to modify.
+<4> The new value to replace the target label.
+<5> The relabel action that replaces the old value with the new value based on regex matching.
+The default action is `Replace`.
+Other possible values are `Keep`, `Drop`, `HashMod`, `LabelMap`, `LabelDrop`, and `LabelKeep`.
+The `Drop` action prevents a firing alert from being delivered to Alertmanager and, as a result, stops any notification routing for this alert.
```

Comment:
I think it's no need to explain `Drop` action separately, we also have other possible actions but not explained here

Reply:
I'd leave it up to the professional writers whether this makes sense here or only adds confusion. We mention this behavior in another place: https://github.com/openshift/openshift-docs/pull/48754/files#diff-e1e506f67411279fb9dd1d14e895b35f0b3ab44af9774073cb306c120da16561R27

The motivation was to mention the somewhat strange behavior of `Drop` when it comes to the alert being shown in the UI, but not routed to notification channels.
But I agree its slightly awkward here, that is my bad.

Reply:
I think @juzhao is correct. It can safely be omitted here since it is mentioned elsewhere and because we're not defining the other options anyway.